client/templates/selectedProduct/selectedProduct.html

```diff
@@ -0,0 +1,30 @@
+<template name="selectedProduct">
+  <h1>Selected Product:</h1>
+    {{#if Template.subscriptionsReady}}
+      {{#if selectedProduct}}
+        {{#with selectedProduct}}
+          {{>product}}
+        {{/with}}
+      {{else}}
+        Sorry, no product with that code...
+      {{/if}}
+    {{else}}
+      Loading...
+    {{/if}}
+</template>
+
+<template name="product">
```

Comment:
Change all the labels to their "friendly names". e.g. <p>itemLookUpCode: {{itemLookUpCode}}</p> to <p>Look Up Code: {{itemLookUpCode}}</p> and <p>reorderPoint: {{reorderPoint}}</p> to <p>Reorder Point: {{reorderPoint}}</p>